fastapi/openapi/models.py

```diff
@@ -11,7 +11,7 @@
     import email_validator
 
     assert email_validator  # make autoflake ignore the unused import
-    from pydantic.types import EmailStr  # type: ignore
+    from pydantic.types import EmailStr
```

Comment:
This `# type: ignore` is not necessary for mypy to pass; let me know if you want this in a separate PR